### PR TITLE
Delete the ABTestController and its state

### PR DESCRIPTION
**Pull request by whymarrh**
This PR removes the `ABTestController` which is currently unused.

If/when we choose to run A/B tests in the future this could be re-added but more than likely we'll need to take on a new implementation.

### CODE REVIEW

File: test/unit/migrations/046-test.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  it('should do nothing if mkrMigrationReminderTimestamp state does not exist', async function () {
+    const oldStorage = {
+      meta: {},
+      data: {
+        AppStateController: {
+          bar: 'baz',
+        },
+        foo: 'bar',
+      },
+    }
```
Comment (rekmarks) [inline test/unit/migrations/046-test.js:47]: Is this old copypasta?

Comment (whymarrh) [reply test/unit/migrations/046-test.js:47]: Yes 😅 but technically it's valid

Comment (whymarrh) [reply test/unit/migrations/046-test.js:47]: Though the description is invalid 🤦 whoops

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Pull request merged**